### PR TITLE
ref(ts): Refactor LoadingError

### PR DESCRIPTION
**Pull request by MeredithAnya**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/loadingError.tsx [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import classNames from 'classnames';
 import {t} from 'app/locale';
 
-class LoadingError extends React.Component {
+type Props = {
+  onRetry?: () => void;
+  message?: string;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/loadingError.tsx]: I think we want the equivalent of `PropTypes.node` (idk what it is, cc @dashed), and it should probably be required

File: src/sentry/static/sentry/app/components/loadingError.tsx [hunk 2]
```diff
@@ -18,9 +21,9 @@ class LoadingError extends React.Component {
   }
 
   render() {
-    const {className, message, onRetry} = this.props;
+    const {message, onRetry} = this.props;
     return (
-      <div className={classNames('alert alert-error alert-block', className)}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/loadingError.tsx]: We probably want to keep this so it can be styled

Comment (MeredithAnya) [reply src/sentry/static/sentry/app/components/loadingError.tsx]: I didn't see any place where we passed in `className`

File: src/sentry/static/sentry/app/components/loadingError.tsx (hunk 1 above)
Comment (MeredithAnya) [reply src/sentry/static/sentry/app/components/loadingError.tsx]: hmm we def just have `return <LoadingError />;`  in places

Comment (billyvg) [reply src/sentry/static/sentry/app/components/loadingError.tsx]: AH ok

File: src/sentry/static/sentry/app/components/loadingError.tsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/loadingError.tsx]: OK, if we don't re-style it, then 👍 	on removing it

File: src/sentry/static/sentry/app/components/loadingError.tsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/loadingError.tsx]: `React.ReactNode` is the type

**Review by billyvg — APPROVED** [review]

**Pull request merged**